roachtest: gorm failed

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5645962?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5645962?buildTab=artifacts#/gorm) on master @ [7a8cac9b11036dcf401453eb36e531fe381b70b4](https://github.com/cockroachdb/cockroach/commits/7a8cac9b11036dcf401453eb36e531fe381b70b4):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1195-g7a8cac9b11
		Tests run against gorm v1.23.7
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-experience
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5658707?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5658707?buildTab=artifacts#/gorm) on master @ [33d70998719051ee058bc9e516afa238ea7b7451](https://github.com/cockroachdb/cockroach/commits/33d70998719051ee058bc9e516afa238ea7b7451):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1232-g33d7099871
		Tests run against gorm v1.23.7
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83797 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1 release-blocker]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5672143?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5672143?buildTab=artifacts#/gorm) on master @ [03d5260a467b8640de41ee08054c1ce05f91966c](https://github.com/cockroachdb/cockroach/commits/03d5260a467b8640de41ee08054c1ce05f91966c):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1281-g03d5260a46
		Tests run against gorm v1.23.7
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83885 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83797 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5685337?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5685337?buildTab=artifacts#/gorm) on master @ [88789feea153b61b379c0240a07482bdfaf0c932](https://github.com/cockroachdb/cockroach/commits/88789feea153b61b379c0240a07482bdfaf0c932):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1334-g88789feea1
		Tests run against gorm v1.23.8
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83885 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83797 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5695598?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5695598?buildTab=artifacts#/gorm) on master @ [54b4fcaa231abd16260fb09722ed291dbd6a4900](https://github.com/cockroachdb/cockroach/commits/54b4fcaa231abd16260fb09722ed291dbd6a4900):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1365-g54b4fcaa23
		Tests run against gorm v1.23.8
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83885 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83797 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5701221?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5701221?buildTab=artifacts#/gorm) on master @ [6374bd83f192bd509e50313acb431690b9c3f8ae](https://github.com/cockroachdb/cockroach/commits/6374bd83f192bd509e50313acb431690b9c3f8ae):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1369-g6374bd83f1
		Tests run against gorm v1.23.8
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83885 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83797 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5706631?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/5706631?buildTab=artifacts#/gorm) on master @ [86e007dcb5cbde3501f138c1d768519db3487857](https://github.com/cockroachdb/cockroach/commits/86e007dcb5cbde3501f138c1d768519db3487857):


```
test artifacts and logs in: /artifacts/gorm/run_1
	orm_helpers.go:191,orm_helpers.go:117,java_helpers.go:220,gorm.go:127,test_runner.go:896: 
		Tests run on Cockroach v22.2.0-alpha.00000000-1371-g86e007dcb5
		Tests run against gorm v1.23.8
		1 Total Tests Run
		0 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: tests.[build failed] - unknown (unexpected)
		For a full summary look at the gorm artifacts 
		
		An updated blocklist (gormBlocklist22_2) is available in the artifacts' gorm log
```
<p>Parameters: <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #83885 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-21.2]
- #83797 roachtest: gorm failed [C-test-failure O-roachtest O-robot T-sql-experience branch-release-22.1]
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Ran the test manually with roachprod and got this error:
```
ubuntu@jane-1657557696-01-n1cpu4-0001:~$ cd /mnt/data1/go/src/github.com/go-gorm/gorm/tests/ && GORM_DIALECT=“postgres” GORM_DSN=“user=root password= dbname=gorm host=localhost port=26257 sslmode=disable” go test -v ./...
# gorm.io/driver/mysql
/home/ubuntu/go/pkg/mod/gorm.io/driver/mysql@v1.3.5/migrator.go:263:52: undefined: gorm.Index
/home/ubuntu/go/pkg/mod/gorm.io/driver/mysql@v1.3.5/migrator.go:264:20: undefined: gorm.Index
/home/ubuntu/go/pkg/mod/gorm.io/driver/mysql@v1.3.5/migrator.go:276:16: undefined: migrator.Index
# gorm.io/driver/postgres
/home/ubuntu/go/pkg/mod/gorm.io/driver/postgres@v1.3.8/migrator.go:617:52: undefined: gorm.Index
/home/ubuntu/go/pkg/mod/gorm.io/driver/postgres@v1.3.8/migrator.go:618:20: undefined: gorm.Index
/home/ubuntu/go/pkg/mod/gorm.io/driver/postgres@v1.3.8/migrator.go:628:16: undefined: migrator.Index
FAIL    gorm.io/gorm/tests [build failed]
FAIL

```
Could this be ralated? https://gorm.io/docs/v2_release_note.html

--- a/pkg/cmd/roachtest/tests/gorm.go
+++ b/pkg/cmd/roachtest/tests/gorm.go
@@ -24,7 +24,7 @@ import (
 )
 
 var gormReleaseTag = regexp.MustCompile(`^v(?P<major>\d+)\.(?P<minor>\d+)\.(?P<point>\d+)$`)
-var gormSupportedTag = "v1.23.5"
+var gormSupportedTag = "v1.23.8"
 
 func registerGORM(r registry.Registry) {
 	runGORM := func(ctx context.Context, t test.Test, c cluster.Cluster) {

